how-to/develop-with-buildd.rst

```diff
@@ -169,20 +169,43 @@ You probably want the next section (:ref:`Configuring Launchpad <configuring-lau
 Getting a base image
 --------------------
 
-First, we need a base image to use for the builds. Usually this is pulled as part of a build, but if we don't have Launchpad involved, we need to set this up manually.
+First, we need a base image to use for the builds. Usually, this is pulled as part of a build, but if we don't have Launchpad involved, we need to set this up manually.
+To download the image we need to set up the ``ubuntu-archive-tools``.
 
 .. code-block:: sh
 
     $ git clone https://git.launchpad.net/ubuntu-archive-tools
     $ sudo apt install python3-launchpadlib python3-ubuntutools
+
+Now we can download the image, note that there are two types of images: ``chroot`` and ``lxd`` images.
```

Comment:
Consider these 2 punctuation changes (full-stop and colon) and say "**all** other build types" to be more explicit.

> Now we can download the image. Note that there are two types of images: ``chroot`` and ``lxd`` images: ``chroot`` backend is only used for ``binarypackagebuilds`` and ``sourcepackagerecipebuilds`` while **all** other build types use a LXD image.